reference/commands/config.rst

```diff
@@ -202,14 +202,20 @@ Displays all the Conan built-in configurations. There are 2 groups:
     core.package_id:default_non_embed_mode: By default, 'minor_mode'
     core.package_id:default_python_mode: By default, 'minor_mode'
     core.package_id:default_unknown_mode: By default, 'semver_mode'
+    core.sources:download_cache: Folder to store the sources backup
+    core.sources:download_urls: List of URLs to download backup sources from
+    core.sources:upload_url: Remote URL to upload backup sources to
```

Comment:
There were not defined on purpose, to not release yet the backup sources, but I am totally fine with it, they can be added here for completitude.